File: how-many-points-did-the-teams-from-los-angeles-score/how-many-points-did-the-teams-from-los-angeles-score.js
function getLosAngelesPoints(results) { 
  var s = /^Los Angeles [A-Z][a-z]*/i;
  var b = results.filter(a=>s.exec(a[0])).reduce((a,e) => a + Number(e[1].split(':')[0]),0)
  return b
}

File: how-many-points-did-the-teams-from-los-angeles-score/how-many-points-did-the-teams-from-los-angeles-score.py
import re
def get_los_angeles_points(results):
    s = " 1234 ".join(["6789".join(i) for i in results])
    t = re.findall(r' 1234 Los Angeles [A-Z][a-z]+6789(\d*)',s)
    print(s,t)
    sum = 0
    for i in t:
        if i:
            sum += int(i)
    return sum